code/modules/projectiles/guns/specialist.dm

```diff
@@ -1010,7 +1010,7 @@
 	max_shells = 4 //codex
 	caliber = CALIBER_ROCKETARRAY //codex
 	load_method = MAGAZINE //codex
-	current_mag = /obj/item/ammo_magazine/rocket/m57a4
+	current_mag = /obj/item/ammo_magazine/rocket/m57a4/ds
```

Comment:
defaulting to deathsquad version is just asking for a funny oversight down the road
```suggestion
	current_mag = /obj/item/ammo_magazine/rocket/m57a4
```